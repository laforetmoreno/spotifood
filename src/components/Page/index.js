import React from 'react';
import { node } from 'prop-types';

import styles from './index.module.scss';

const Page = ({ children }) => <main className={styles.wrapper}>{children}</main>;

Page.propTypes = {
  children: node,
};

Page.defaultProps = {
  children: null,
};

export default Page;
