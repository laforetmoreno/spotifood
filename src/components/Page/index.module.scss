@import '../../styles';

.wrapper {
  padding: 0 rem(20px) rem(20px) rem(20px);
  background: $--background-default;
  min-height: calc(100vh - 40px);
}
