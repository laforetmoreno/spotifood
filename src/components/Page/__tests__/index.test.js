import React from 'react';
import renderer from 'react-test-renderer';
import Page from '../index';

describe('<Page />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Page>
          <h1>test</h1>
        </Page>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
