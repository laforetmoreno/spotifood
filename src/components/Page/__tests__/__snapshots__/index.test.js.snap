// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> renders correctly 1`] = `
<main
  className="wrapper"
>
  <h1>
    test
  </h1>
</main>
`;
